### ISSUE TITLE
Support for Svelte 5

### ISSUE THREAD

**Issue by elivibe** [SikandarJODD/svelte-animations#33]
hi @SikandarJODD 

Is there a plan to support [Svelte 5](https://github.com/sveltejs/svelte/milestone/9)? The API is currently stabilizing, and a migration tool has recently been released.

https://svelte-5-preview.vercel.app/docs/old-vs-new
https://x.com/dummdidumm_/status/1844750782647685151

**Comment by SikandarJODD** [comment]
yes yes sureeee i am working on it - it's hard with svelte-motion lib - i will try to convert rest of the components in svelte 5

**Comment by cliffordkleinsr** [comment]
We're working on an advancement lib of svelte-motion, This includes an inbuilt typescript and will bump framer motion & svelte5 support. See [here](https://github.com/micha-lmxt/svelte-motion/issues/36) for the conversation thread and [here](https://www.npmjs.com/package/motion-start) for the library

**Comment by SikandarJODD** [comment]
invalid link : discord 
![image](https://github.com/user-attachments/assets/ba2e6c12-e959-43f2-8dde-0f3d7cf562a2)
we can connect on twitter : https://x.com/Sikandar_Bhide

**Comment by cliffordkleinsr** [comment]
Woops sorry  about that here's a fresh link https://discord.gg/r6tSZjdR

**Comment by devNamanG** [comment]
First of all, thank you for making such a good library. So, does the library support svelte 5 now? or is the work in progress? Any expected timeline for when the work might be complete?

**Comment by SikandarJODD** [comment]
Yeep i am waiting for good Animation lib for svelte 5...for now i will create all other animation which doesn't include svelte motion

**Comment by devNamanG** [comment]
Ohh, that is good to know, thank you again! :)

**Comment by julien-blanchon** [comment]
What about motion.js (https://motion.dev/docs/press?platform=js) with Svelte action ?

**Comment by SikandarJODD** [comment]
Yeep good point but i don't like : source code hidden : 
it makes so hard to learn and build stuff using it ..

<img width="667" height="618" alt="Image" src="https://github.com/user-attachments/assets/10e88a09-0fc0-40f2-80b1-42d445dd6168" />

**Comment by julien-blanchon** [comment]
Yeah it's annoying but it's the best animation library soo far. 
I like that it use css transition api when possible
